### DISCUSSION TITLE
How do you disable/move the floating text: "Press ESC to show your cursor"

### DISCUSSION THREAD

**Discussion by markwong167** [jeffreylanters/react-unity-webgl#292] [Help Wanted!] [answered]
Hey all, I've been having trouble finding out how to disable/move the floating text/popup: "Press ESC to show your cursor" when you lock the cursor.
![image](https://user-images.githubusercontent.com/19193927/147765128-fc6df59f-7c48-4a4e-9efd-dbfdea708233.png)

**Comment by jeffreylanters** [comment, marked as answer]
Hi! I'm not sure, but this looks like something implemented natively in your browser? I'm not sure if you can disable this behaviour, but if you can, it's not related to this module.

**Reply by markwong167** [reply]
That helps a lot, thanks!!!
